.buildkite/build_windows_installer.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+PARENT_PATH=$(pwd)
+KOLIBRI_DOCKER_PATH="$PARENT_PATH/windows_installer_docker_build"
+KOLIBRI_WINDOWS_PATH="$KOLIBRI_DOCKER_PATH/kolibri-installers/windows"
+
+mkdir dist
+buildkite-agent artifact download 'dist/*.whl' dist/
+make writeversion
+
+# Clone kolibri windows installer base in develop branch.
+cd $KOLIBRI_DOCKER_PATH \
+    && git clone https://github.com/learningequality/kolibri-installers.git \
+    && cd kolibri-installers \
+    && git checkout develop
```

Comment:
Maybe it would be smart to simply check this code into the kolibri repo rather than maintaining it separately? What are the pros/cons?